include/git2/diff.h

```diff
@@ -130,7 +130,9 @@ typedef enum {
 	GIT_DIFF_INCLUDE_CASECHANGE = (1u << 11),
 
 	/** If the pathspec is set in the diff options, this flags means to
-	 *  apply it as an exact match instead of as an fnmatch pattern.
```

Comment:
This is a rather breaking change. Can we add another switch which changes it to prefix rather together with the disabling version?